tests/cases/example-two-fer/expected/representation.txt

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
An off-topic suggestion: maybe the output should be trimmed?

Reply:
Any reason that it should be trimmed? It's how nim outputs the representation naturally, so it's independent of the submission's whitespacing. Not sure why it does it though.

Reply:
Oh okay, I sort of assumed it was a mistake :) But if this is the natural representation, ignore my comment!

Reply:
I'm happy to trim it. I would just mean removing one character from every representation. 

This can be tested with this macro:
```nim
macro t(code: untyped): untyped =
  echo code.repr

t:
  let x = 1
```

Reply:
I think I would recommend using the default representation, so leaving it as is.

Reply:
Ok great